.gitignore

```diff
@@ -7,3 +7,6 @@ AirpodsPro Battery/Pods/Pods.xcodeproj/xcuserdata/mohamed.xcuserdatad/xcschemes/
 AirpodsPro Battery/.DS_Store
 .DS_Store
 AirpodsPro Battery/.DS_Store
+AirpodsPro Battery/AirpodsPro Battery.xcodeproj/xcuserdata/jay.xcuserdatad/xcschemes/xcschememanagement.plist
+AirpodsPro Battery/Pods/Pods.xcodeproj/xcuserdata/jay.xcuserdatad/xcschemes/xcschememanagement.plist
+AirpodsPro Battery/.DS_Store
```

Comment:
Can we discard those file. This is related to your own environment and not project related. Those files can be ignore in your local gitignore. 

We can also put `*.xcuserdatad/` instead. 

But for now please discard those changes. I will take care of this one.